Sources/DeepLearning/Layer.swift

```diff
@@ -101,6 +101,80 @@ public extension Layer {
     }
 }
 
+extension Differentiable {
```

Comment:
Use `public extension` so you can drop `public`s from individual methods.

Reply:
Thanks; fixed!